srv_fai_config/disk_config/SEAPATH_LVM_EFI

```diff
@@ -6,7 +6,7 @@ disk_config disk1 disklabel:gpt fstabkey:uuid align-at:1M
 
 primary /boot/efi 512M  vfat    rw
 primary /boot	500	ext4	rw,noatime
-primary -       30G	-       -
+primary -       10G	-       -
```

Comment:
This is not linked to the topic at hand in this commit. Did you add this by mistake?

Reply:
yes indeed, 30G was to big for a test on a small vm. That was not present in the initial commit but I added this file by mistake when correcting Matthieu's suggestion.